### DISCUSSION TITLE
Join me on Discord!

### DISCUSSION THREAD

**Discussion by thijsvanloef** [thijsvanloef/palworld-server-docker#132] [Announcements]
I've just opened the Palworld on Docker Discord server!

Here you can:

- Find answers to your server setup questions.
- Chat with other contributors.
- Look for players to join your server.

[Join the discord!](https://discord.gg/UxBxStPAAE)

**Comment by MetraxDev** [comment]
Mind enabling the general channel? :D  Or have some kind of general talk channel?

**Comment by ishkabo** [comment]
Link is currently invalid.

**Reply by ishkabo** [reply]
@thijsvanloef bump! link is non-valid at this time.

**Reply by thijsvanloef** [reply]
Here is another link: https://discord.gg/CWM47StXV

**Reply by ishkabo** [reply]
Sadly no, that one doesn't work either. Thank you for your efforts so far though, I just want to joing to praise you, the image works great for me and has so many features I value like auto pause and auto update.

<img width="765" height="672" alt="image" src="https://github.com/user-attachments/assets/6564823a-845f-4fa3-99b2-d95cd48b16ad" />
